src/tests/zf_apps/zfsink.c

```diff
@@ -290,6 +347,8 @@ int main(int argc, char* argv[])
   if( cfg_rx_timestamping )
     ZF_TRY(zf_attr_set_int(attr, "rx_timestamping", 1));
 
+  ZF_TRY((res = calloc(1, sizeof(*res))) != NULL);
```

Comment:
Given this is just a small test app, I think we could just have a single global instance of struct resources rather than dynamically allocating and passing around the functions.